pages/translations/en/LC_MESSAGES/messages.po

```diff
@@ -14,6 +14,10 @@ msgstr ""
 msgid "AMP Brand Materials"
 msgstr ""
 
+#: /views/partials/footer.j2:85
+msgid "&copy; 2019 AMP Open Source Project"
```

Comment:
I don't think we need to translate this

Reply:
Ah ok, I thought Open Source Project was something that would make sense to localize. will remove